tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -750,11 +750,11 @@ def is_screen_enabled(self):
 
     def touch_home_button(self):
         apps = GaiaApps(self.marionette)
-        if apps.displayed_app.name.lower() != 'default home screen':
+        if apps.displayed_app.manifest_url != 'app://verticalhome.gaiamobile.org/manifest.webapp':
             # touching home button will return to homescreen
             self._dispatch_home_button_event()
             Wait(self.marionette).until(
-                lambda m: apps.displayed_app.name.lower() == 'default home screen')
+                lambda m: apps.displayed_app.manifest_url == 'app://verticalhome.gaiamobile.org/manifest.webapp')
```

Comment:
I tried with Homescreen(self.marionette).manifest_url and when I run a test I get a failure:
  if foundpath == filepath:
TEST-UNEXPECTED-ERROR | test_marketplace_packaged_app.py TestSearchMarketplaceAndInstallApp.test_search_and_install_app | NameError: global name 'Homescreen' is not defined

Traceback (most recent call last):
  File "/home/mermi/gaia/tests/python/gaia-ui-tests/myenv/local/lib/python2.7/site-packages/marionette_client-0.17-py2.7.egg/marionette/marionette_test.py", line 296, in run
    testMethod()
  File "/home/mermi/gaia/tests/python/gaia-ui-tests/gaiatest/tests/functional/marketplace/test_marketplace_packaged_app.py", line 39, in test_search_and_install_app
    self.device.touch_home_button()
  File "/home/mermi/gaia/tests/python/gaia-ui-tests/gaiatest/gaia_test.py", line 753, in touch_home_button
    if apps.displayed_app.manifest_url != Homescreen(self.marionette).manifest_url:
TEST-INFO took 92783ms
so for that I make the url directly